src/Data/Functor/Rep.hs

```diff
@@ -612,6 +612,17 @@ instance Representable f => Applicative (Co f) where
   pure = pureRep
   (<*>) = apRep
 
+  -- | This method is /O(1)/. 'Representable' functors are isomorphic
+  -- to functions and '<*' for functions drops its second argument and
+  -- returns its first.
+  as <* _  = as
+  -- | This method is /O(1)/. 'Representable' functors are isomorphic
+  -- to functions and '*>' for functions drops its first argument and
+  -- returns its first.
```

Comment:
Typo:

```suggestion
  -- returns its second.
```

Reply:
Addressed in 7d0c1f9185e064b476ae6ee86df7c79f2101611b.